base/applications/mspaint/canvas.cpp

```diff
@@ -208,6 +211,10 @@ VOID CCanvasWindow::DoDraw(HDC hDC, RECT& rcClient, RECT& rcPaint)
     // Clean up hdcMem0
     ::SelectObject(hdcMem0, hbm0Old);
     ::DeleteDC(hdcMem0);
+    ::DeleteObject(hbmCache1);
+    ::DeleteObject(hbmCache2);
```

Comment:
Suggestion to clean up in the reverse order of the allocations.
```suggestion
    ::DeleteObject(hbmCache2);
    ::DeleteObject(hbmCache1);
```